site/content/m3query/config/_index.md

```diff
@@ -14,27 +14,27 @@ By default M3 runs two query engines:
 
 Prometheus Query Engine is the default one when calling query endpoint:
```

Comment:
I can't really find this page in the preview for some reason. I tried searching for "But you can switch between the two" and the search sends me [here](https://deploy-preview-3281--m3db.netlify.app/docs/m3query/config/), but the page looks different.